packages/ui-components/style/toolbar.css

```diff
@@ -101,6 +102,11 @@ button.jp-ToolbarButtonComponent .jp-ToolbarButtonComponent-label {
   font-family: var(--jp-ui-font-family);
 }
 
+.jp-ToolbarLabelComponent {
+  background-color: var(--jp-brand-color1);
+  color: white;
```

Comment:
Does it look good in both light and dark mode? Can you add screenshots to the top-level comment?

I have not tried it yet as the Binder is flaky.

Reply:
No, I'll give that a try and post the screen shots

Reply:
I updated this to `var(--jp-ui-inverse-font-color1)`. Let me know what you think. I added in screenshots to the top-level-comment that you can see.